code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -152,7 +152,8 @@
 					/obj/item/device/oxycandle = 1,
 					/obj/item/device/flashlight/flare/glowstick/red = 1,
 					/obj/item/stack/medical/bruise_pack = 1,
-					/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 1
+					/obj/item/reagent_containers/hypospray/autoinjector/inaprovaline = 1,
+					/obj/item/reagent_containers/hypospray/autoinjector/dylovene = 1,
```

Comment:
I'm concerned this is too much of a power boost. Won't it make toxin use difficult for antagonists? And it will let people make tricordizine in their blood which is quite good.

@NonQueueingMatt might comment later about it.

Reply:
I hadn't considered the tricordrazine aspect of giving everyone inaprovaline and dylovene - good point.

I'll remove dylovene from the emergency boxes, though I'll keep an eye out on any instances of antagonists using poisons (especially during rounds without Medical). The dylovene that's now in the white first aid kits should be enough though, I reckon.